script/update-tasks.sh

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
```

Comment:
one more thought if we keep update-tasks.sh directly under script i believe users will thinks its common to k8s and Openshift

shall we create a folder `Openshift` inside script ?
WDYT ?

Reply:
i agree.
In addition, let us keep this script in in `hack/openshift` instread of creating a `scripts` dir.

(the scripts directory existed in the old repository instread of hack directory. as we have a hack dir in this repo, let us use it.)

Reply:
:+1: